domains/pritesh.json

```
@@ -4,6 +4,9 @@
     "email": "epritesh@gmail.com"
   },
   "records": {
-    "CNAME": "personal-website-ochre-mu-51.vercel.app"
+    "A": ["76.76.21.21"],
+    "TXT": [
+      "_vercel=vc-domain-verify=pritesh.is-a.dev,a297d9122cd5f6fad482"
+    ]
```

check the step 5 -> https://docs.is-a.dev/guides/vercel/

you need to also create _vercel.pritesh.json and put the TXT records there
```suggestion
    "A": ["76.76.21.21"]
```